modules/apps/portal/portal-upgrade-test/src/testIntegration/java/com/liferay/portal/upgrade/v7_0_1/test/UpgradeDocumentLibraryTest.java

```diff
@@ -0,0 +1,226 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.upgrade.v7_0_1.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.document.library.kernel.processor.RawMetadataProcessor;
+import com.liferay.portal.events.StartupHelperUtil;
+import com.liferay.portal.kernel.dao.jdbc.DataAccess;
+import com.liferay.portal.kernel.test.ReflectionTestUtil;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.version.Version;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.tools.DBUpgrader;
+import com.liferay.portal.upgrade.PortalUpgradeProcess;
+import com.liferay.portal.upgrade.v7_0_1.UpgradeDocumentLibrary;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.apache.commons.lang.time.StopWatch;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Alicia García
+ */
+@RunWith(Arquillian.class)
+public class UpgradeDocumentLibraryTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new LiferayIntegrationTestRule();
+
+	@BeforeClass
+	public static void setUpClass() {
+		_originalStopWatch = ReflectionTestUtil.getAndSetFieldValue(
+			DBUpgrader.class, "_stopWatch", null);
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+		ReflectionTestUtil.setFieldValue(
+			DBUpgrader.class, "_stopWatch", _originalStopWatch);
+	}
+
+	@Before
+	public void setUp() throws Exception {
+		try (Connection connection = DataAccess.getConnection()) {
+			_currentSchemaVersion =
+				PortalUpgradeProcess.getCurrentSchemaVersion(connection);
+
+			_updateSchemaVersion(connection, _ORIGINAL_SCHEMA_VERSION);
+
+			_upgrading = StartupHelperUtil.isUpgrading();
+		}
+	}
+
+	@After
+	public void tearDown() throws Exception {
+		try (Connection connection = DataAccess.getConnection()) {
+			_updateSchemaVersion(connection, _currentSchemaVersion);
+		}
+	}
+
+	@Test
+	public void testUpgrade() throws Exception {
+		long classNameId = RandomTestUtil.randomLong();
+
+		Connection connection = DataAccess.getConnection();
+
+		long originalClassNameId = _getClassNameIdFromClassName(
```

Comment:
@AliciaGarciaGarcia, className will always exist in this version of the portal, so we can skip the checking

Reply:
Thanks

I will remove the conditionals

<img src="https://media3.giphy.com/media/xT0xelpBUyVmjxqGTC/giphy.gif"/>